examples/run/task-run-resource-spec.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: pipeline.knative.dev/v1alpha1
+kind: TaskRun
+metadata:
+  name: read-file
```

Comment:
could you add a brief description of this example to https://github.com/knative/build-pipeline/tree/master/examples#examples ?

(looks like it's pretty much the same as [the existing run example](https://github.com/knative/build-pipeline/tree/master/examples#simple-runs) but with resources embedded?)

Reply:
I did not know this file existed. 👍

Reply:
I think I have a tendency to put READMEs in each folder, and you tend to look for them in `docs/` - maybe we can find some happy middle ground XD 😅